style.css

```diff
@@ -0,0 +1,62 @@
+.inputish {
+  width: 30rem;
+  margin-left: 80px;
+  margin-right: 0%;
+  height: 2rem;
+  margin-bottom: 2%;
+  margin-top: 10px;
+  vertical-align: middle;
+}
+
+.divish {
```

Comment:
This can be specified in pixel or reduced in its current unit, to avoid the mess it is causing now